packages/plugin-infra/src/types.ts

```diff
@@ -597,14 +597,15 @@ export namespace Plugin.SNSAdaptor {
              */
             TabContent: InjectUI<{
                 identity?: SocialIdentity
-                socialAddressList?: Array<SocialAddress<NetworkPluginID>>
+                persona?: string
+                socialAddress?: SocialAddress<NetworkPluginID>
             }>
         }
         Utils?: {
             /**
              * If it returns false, this tab will not be displayed.
              */
-            shouldDisplay?(identity?: SocialIdentity, addressNames?: Array<SocialAddress<NetworkPluginID>>): boolean
+            shouldDisplay?(identity?: SocialIdentity, addressName?: SocialAddress<NetworkPluginID>): boolean
```

Comment:
Why this change?

Reply:
component will receive only one address now